packages/generator-one-app-module/__tests__/app.js

```diff
@@ -136,7 +136,7 @@ describe('generator-one-app-module', () => {
       assert.jsonFileContent('package.json', {
         'one-amex': {
           bundler: {
-            requiredExternals: ['react-intl'],
+            externals: 'react-intl',
```

Comment:
This is wrong? Should be an array no? And should be named `requiredExternals`.

Reply:
Sorry @anescobar1991 I was testing something, @NarmeenNaveedAhmed has the official PR I will delete this one

Reply:
if you could review this one instead please https://github.com/americanexpress/one-app/pull/82 😅